public/data/githubarj.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "RICHARD JEREMY GITHUBA",
+  "type": "personal",
+  "bio": "Full Stack Software Developer",
+  "avatar": "https://github.com/githubarj.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/githubarj",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/richard-jeremy-githuba-80548922a/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/githubarj",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+        {
+      "title": "Learning to Code!",
+      "date": "February 2022",
+      "icon": "github",
+      "color": "blue",
+      "description": "Started learning to code with the freeCodeCamp curriculum!",
+      "url": "https://www.freecodecamp.org/learn"
+    },
+    {
+      "title": "Started open Source",
+      "date": "August 2020",
+      "icon": "github",
+      "color": "grey",
+      "description": "Started Contributing to Open Source",
+      "url": "https://github.com/githubarj"
+    }
+  ]
+}
```

Just a endline is missing :D 
```suggestion
}

```